tools/report.sh

```diff
@@ -48,14 +44,20 @@ while getopts "$optspec" optchar; do
                     usage
                     ;;
             esac;;
-        *)
-            cluster="my-cluster"
-            namespace="myproject"
-            ;;
     esac
 done
 shift $((OPTIND-1))
 
+if [ -z $cluster ]; then
```

Comment:
Could we join this into a single if? So that the user who doesn't specify either gets both at the same time?